### DISCUSSION TITLE
Add Sub-Category Menus for Artists and Songs Like There Already Is For Albums

### DISCUSSION THREAD

**Discussion by dathbe** [navidrome/navidrome#2746] [Ideas]
The dropdown selections for Albums (Random, Favourites, Top Rated, Recently Added, Recently Played, and Most Played) are great, but I tend to view my music as songs and not Albums.  Thus, I would love to see the same sub-menus for Artists and Songs so that you can, for example, view your Recently Added songs even if you've had other songs from the album for a while.  
![2023-12-25 16_16_37-Navidrome — Mozilla Firefox Private Browsing](https://github.com/navidrome/navidrome/assets/14321825/02eab83a-4944-4e8c-87af-6a5b07aafc7b)

**Comment by deluan** [comment]
This has been proposed already. We are rewriting the UI, so a new filtering/sorting system will be implemented. Thanks for the suggestion anyway.

c/c @caiocotts

**Comment by dathbe** [comment]
Thanks!
